src/indexer/repository/transfer.ts

```diff
@@ -107,6 +111,14 @@ class TransferRepository {
         },
       },
       timestamp: decodedLog.timestamp,
+      account: {
+        create: {
```

Comment:
You shouldn't create a new account in `updateTransfer` function

Reply:
done